node/src/Commands.ts

```diff
@@ -2407,17 +2417,36 @@ export function createXRead(
     keys_and_ids: Record<string, string>,
     options?: StreamReadOptions,
 ): command_request.Command {
-    const args: string[] = [];
+    let args: string[] = [];
```

Comment:
nit: 
```suggestion
    let args: string[] = addReadOptions(options);
```

then remove the next condition